src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,48 @@
+import { Card } from "shoto-ui";
+import "./carousel.css";
+
+export const Carousel = () => {
+  return (
+    <div className="container-carousel">
+      <Card

```

Comment:
You can use a `map` higher-order function here. Put all the details into an array of objects. A sample object structure is 
```
{
        heading:"Cyberpunk 2077",
        price:"₹ 2,999",
        description:"Cyberpunk 2077 is an open-world story set in Night City.",
        tags:["PC", "Playstation", "Xbox"],
        actionBtnText:"Buy Now",
        action:"Pre Order",
        bgImage:"https://c4.wallpaperflare.com/wallpaper/39/346/426/digital-art-men-city-futuristic-night-hd-wallpaper-preview.jpg"
}
```